### PR TITLE
fix: updated path for ean8 and short barcodes (padding with zeroes)

### PR DESCRIPTION
**Pull request by stephanegigandet**
Please see #3818 for details

### CODE REVIEW

File: lib/ProductOpener/Products.pm [hunk 1]
```diff
@@ -75,6 +75,7 @@ BEGIN
 		&server_for_product_id
 		&data_root_for_product_id
 		&www_root_for_product_id
+		&split_code
```
Comment (hangy) [inline lib/ProductOpener/Products.pm:78]: It's already exported here: https://github.com/openfoodfacts/openfoodfacts-server/blob/89119c8f92ee67120b95f1037c8ad453c1929a55/lib/ProductOpener/Products.pm#L73

Comment (stephanegigandet) [reply lib/ProductOpener/Products.pm:78]: Thanks

**Comment by stephanegigandet** [comment]
> Do the images need to be moved, too?

Yes, I forgot to mark the PR as a draft, I have not finished.

**Comment by teolemon** [comment]
@stephanegigandet merge conflict on the test before a rebase is possible

**Comment by stephanegigandet** [comment]
I will make a new PR.

**Pull request closed without merging**